gatekeeper-3.13.yaml

```diff
@@ -38,6 +41,9 @@ subpackages:
       - runs: |
           mkdir -p ${{targets.subpkgdir}}
           ln -sf /usr/bin/manager ${{targets.subpkgdir}}/manager
+    dependencies:
+      provides:
+        - gatekeeper-compat=3.13
```

Comment:
Same re: versioned `${{package.name}}` for subpackage names and `.999` here

Reply:
Done